data/json/requirements/ammo.json

```diff
@@ -34,6 +34,12 @@
     "//": "Components required for .357 Magnum ammo",
     "components": [ [ [ "357mag_casing", 1 ] ], [ [ "lgpistol_primer", 1 ] ] ]
   },
+  {
+    "id": "ammo_357sig",
+    "type": "requirement",
+    "//": "Components required for .357 SIG ammo",
+    "components": [ [ [ "357sig_casing", 1 ] ], [ [ "lgpistol_primer", 1 ] ] ]
```

Comment:
.357 SIG uses small pistol primers.

Reply:
It does? Thought it'd be large like the .357 Magnum it was designed to compete with.